ols/constants.py

```diff
@@ -13,14 +13,17 @@
 
 # templates
 SUMMARIZATION_TEMPLATE = """
-The following context contains several pieces of documentation. Please answer the user's question based on this context.
-Documentation context:
-{context_str}
+Instructions:
+- You are an expert in Kubernetes and OpenShift.
+- Respond to questions about topics other than Kubernetes and OpenShift with: "I can only answer questions about Kubernetes and OpenShift"
+- Refuse to participate in anything that could harm a human.
+- Use the previous chat history, and the context, to interact and help the user."
```

Comment:
Is 
_You are an expert in Kubernetes and OpenShift._
instruction necessary? What is its outcome? Some confident tone in conversation?

Similarly the 3. 4. instruction. Aren't those encoded in the llms already?

Reply:
not just tone, it frames the whole way the LLM attempts to answer the question and what knowledge it brings to bear on the problem.  It influences the llm to take on that persona and bring specific expertise to the conversation.

Reply:
I dont think I have to change anything here @onmete . Please confirm.

Reply:
Yes, I'll try to poke in the area, but no changes are needed here.